Modules/Loadable/Markups/MRML/vtkMRMLMarkupsNode.cxx

```diff
@@ -509,45 +507,29 @@ int vtkMRMLMarkupsNode::AddMarkupWithNPoints(int n, std::string label)
   Markup markup;
   markup.Label = label;
   this->InitMarkup(&markup);
-  for (int i = 0; i < n; i++)
+  if (point!=NULL)
```

Comment:
I use spaces in my for statements, please add them here to keep the file consistent with the rest of the module.